urshbase.sql

```diff
@@ -0,0 +1,21 @@
+CREATE DATABASE urshbase;
+USE urshbase;
+
+CREATE TABLE shorten_urls (
+id int(11) NOT NULL AUTO_INCREMENT,
```

Comment:
Please follow the coding standards, I mean just add the 2 spaces in the inner content (code).

In result you will see something like this -
```mysql
CREATE TABLE shorten_urls (
  id int(11) NOT NULL AUTO_INCREMENT,
  code varchar(5) NOT NULL,
  point varchar(256) NOT NULL,
  PRIMARY KEY (id)
);
```